pkg/apis/playlist/v0alpha1/legacy_storage.go

```diff
@@ -109,3 +112,88 @@ func (s *legacyStorage) Get(ctx context.Context, name string, options *metav1.Ge
 
 	return convertToK8sResource(dto, s.namespacer), nil
 }
+
+func (s *legacyStorage) Create(ctx context.Context,
+	obj runtime.Object,
+	createValidation rest.ValidateObjectFunc,
+	options *metav1.CreateOptions,
+) (runtime.Object, error) {
+	info, err := grafanarequest.NamespaceInfoFrom(ctx, true)
+	if err != nil {
+		return nil, err
+	}
+
+	p, ok := obj.(*Playlist)
+	if !ok {
+		return nil, fmt.Errorf("expected playlist?")
+	}
+	cmd, err := convertToLegacyUpdateCommand(p, info.OrgID)
+	if err != nil {
+		return nil, err
+	}
+	out, err := s.service.Create(ctx, &playlist.CreatePlaylistCommand{
+		UID:      p.Name,
+		Name:     cmd.Name,
+		Interval: cmd.Interval,
+		Items:    cmd.Items,
+		OrgId:    cmd.OrgId,
+	})
+	if err != nil {
+		return nil, err
+	}
+	return s.Get(ctx, out.UID, nil)
+}
+
+func (s *legacyStorage) Update(ctx context.Context,
```

Comment:
i'm seeing some issues with `kubectl apply -f playlist.yaml`:
```yaml
apiVersion: playlist.x.grafana.com/v0alpha1
kind: Playlist
metadata:
  name: hi
  namespace: default
spec:
  interval: 10m
  items:
  - type: dashboard_by_uid
    value: bb2e1077-98a7-4755-9700-245d5e5327c4
  title: hi
```

if the resource doesn't exist, i get this:
```
$ kubectl apply -f playlist.yaml
Error from server: error when retrieving current configuration of:
Resource: "playlist.x.grafana.com/v0alpha1, Resource=playlists", GroupVersionKind: "playlist.x.grafana.com/v0alpha1, Kind=Playlist"
Name: "hi", Namespace: "default"
from server for: "playlist.yaml": Playlist not found
```

if it does exist, i get this:
```
kubectl apply -f playlist.yaml
Warning: resource playlists/hi is missing the kubectl.kubernetes.io/last-applied-configuration annotation which is required by kubectl apply. kubectl apply should only be used on resources created declaratively by either kubectl create --save-config or kubectl apply. The missing annotation will be patched automatically.
Error from server: error when applying patch:
{"metadata":{"annotations":{"kubectl.kubernetes.io/last-applied-configuration":"{\"apiVersion\":\"playlist.x.grafana.com/v0alpha1\",\"kind\":\"Playlist\",\"metadata\":{\"annotations\":{},\"name\":\"hi\",\"namespace\":\"default\"},\"spec\":{\"interval\":\"10m\",\"items\":[{\"type\":\"dashboard_by_uid\",\"value\":\"bb2e1077-98a7-4755-9700-245d5e5327c4\"}],\"title\":\"test-hi\"}}\n"}}}
to:
Resource: "playlist.x.grafana.com/v0alpha1, Resource=playlists", GroupVersionKind: "playlist.x.grafana.com/v0alpha1, Kind=Playlist"
Name: "hi", Namespace: "default"
for: "playlist.yaml": error when patching "playlist.yaml": no kind "Playlist" is registered for the internal version of group "playlist.x.grafana.com" in scheme "pkg/runtime/scheme.go:100"
```